### PR TITLE
Some missing translations for Hindi(hi) language have been translated

### PR DESCRIPTION
**Pull request by harishnandan**
The missing translations for Hindi(hi) have been corrected. I will add for one more language soon.

### CODE REVIEW

**Comment by borisyankov** [comment]
@harishnandan the workflow we use for the translations is:
* you do translate the strings within the Transifex system
* we pull down all strings from there and merge to the project

If committing directly to the repo, they will be overwritten on later imports from Transifex.

File: src/i18n/translations/messages_hi.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
   "Why not start the conversation?": "चलो बातचीत शुरू करें?",
   "Chat": "बातचीत शुरू करें",
   "Log in with Google": "Google के साथ प्रवेश करें",
-  "Log in with GitHub": "Log in with GitHub",
+  "Log in with GitHub": "GitHub के साथ प्रवेश करें",
```
Comment (kunall17) [inline src/i18n/translations/messages_hi.json]: Also write the GitHub here in hindi!

File: src/i18n/translations/messages_hi.json [hunk 2]
```diff
@@ -61,7 +61,7 @@
   "starred": "तारांकित",
   "Enable notifications": "सूचना सक्षम करें",
   "Jot down something": "कुछ लिखिए",
-  "Message {recipient}": "संदेश {recipient}",
+  "Message {recipient}": "संदेश {प्राप्त करने वाला}",
```
Comment (kunall17) [inline src/i18n/translations/messages_hi.json]: This shouldn't be translated as its for replacing, and here Message is signifying as a verb to send that person a message, so its better to be written as भेजें

File: src/i18n/translations/messages_hi.json [hunk 3]
```diff
@@ -130,5 +130,5 @@
   "Active": "सक्रिय उपयोगकर्ता",
   "Idle": "दूर उपयोगकर्ताओं",
   "Offline": "ऑफ़लाइन उपयोगकर्ता",
-  "Do not mark messages read on scroll": "Do not mark messages read on scroll"
+  "Do not mark messages read on scroll": "स्क्रॉल पर पढ़े संदेशों को चिह्नित नहीं करें"
```
Comment (kunall17) [inline src/i18n/translations/messages_hi.json]: Don't you think using निशान instead of चिह्नित would be better?

**Comment by harishnandan** [comment]
@kunall17 @borisyankov Thanks for the feedback. I will update the requested changes while uploading the file to Transifex.

File: src/i18n/translations/messages_hi.json (hunk 3 above)
Comment (pulkonet) [reply src/i18n/translations/messages_hi.json]: No. I think चिह्नित is better.

**Comment by borisyankov** [comment]
Thanks @harishnandan
Closing this PR.

**Pull request closed without merging**